event_routing_backends/models.py

```diff
@@ -240,6 +245,26 @@ def _match_event_for_host(self, original_event, host_config):
             bool
         """
         for key, value in host_config.get('match_params', {}).items():
-            if get_value_from_dotted_path(original_event, key) != value:
+            original_event_value = get_value_from_dotted_path(original_event, key)
+            if isinstance(value, list):
+                matched = False
+                for value_item in value:
+                    if original_event_value:
+                        if bool(re.compile(value_item)) and re.search(value_item, original_event_value):
```

Comment:
I think re.compile will throw an exception here if `value_item` is not valid regex. A better way would be do re.compile with a try-catch statement before the if statement for example:
try:
     h = re.compile(value_item)
except:
     log here
else:
       h.match(original_event_value)